### ISSUE TITLE
Multiple Cities

### ISSUE THREAD

**Issue by rudibarani** [xfjx/homebridge-wunderground#8]
I have tried to add multiple cities using different device names. This works without error messages - but all cities seem to have the same values for temperature and humidity. Any idea, what I am doing wrong?

**Comment by xfjx** [comment]
For me this looks like a (display) bug in iOS10. It worked before and I now still get different temperatures for different cities sometimes. Will try to further look into this.

**Comment by rudibarani** [comment]
Did you get a chance to dive on the problem? I was using only one city for some time and tried again to add others today. All cities show identical temperature and humidity values in every Homekit-App I have tried.

**Comment by xfjx** [comment]
Are you on the latest version?

**Comment by rudibarani** [comment]
Yes. I doublechecked and no update is available: homebridge-wunderground@0.0.5
